src/open_inwoner/js/components/anchor-menu/anchor-menu-mobile.js

```diff
@@ -24,3 +24,13 @@ export class AnchorMobile {
  */
 const anchorToggles = document.querySelectorAll(AnchorMobile.selector)
 ;[...anchorToggles].forEach((anchorToggle) => new AnchorMobile(anchorToggle))
+
+/**
+ * Controls the toggling of anchor list specifically for the cases-detail page
+ */
+export class AnchorMobileOOB extends AnchorMobile {}
+
+const anchorTogglesOob = document.querySelectorAll(AnchorMobileOOB.selector)
+;[...anchorTogglesOob].forEach(
+  (anchorToggleOob) => new AnchorMobileOOB(anchorToggleOob)
+)
```

Comment:
I'm not sure what is going on here: the AnchorMobileOOB inherits the selector from AnchorMobile so in this file you'd hit the same nodes twice. 

And in the hydration part this would do the same thing as before this PR (because it is the same code running from the same selector).